File: module-domain/src/main/java/eu/geekhome/services/configurable/DoubleField.java
package eu.geekhome.services.configurable;

import eu.geekhome.services.localization.Resource;

public class DoubleField extends FieldDefinition<Double> {

    public DoubleField(String name, Resource hint, int maxSize, Validator<Double>... validators) {
        super(name, hint, maxSize, Double.class, new DoubleFieldBuilder(), validators);
    }
}

File: module-domain/src/main/java/eu/geekhome/domain/configurable/DoubleField.kt
package eu.geekhome.domain.configurable;

import eu.geekhome.data.localization.Resource;

public class DoubleField extends FieldDefinition<Double> {

    public DoubleField(String name, Resource hint, int maxSize, Validator<Double>... validators) {
        super(name, hint, maxSize, Double.class, new DoubleFieldBuilder(), validators);
    }
}
